Tests/Unit/WhiteSpace/PHPOpenTagSniffTest.phptest

```diff
@@ -0,0 +1,9 @@
+<?php echo'foo'?>
+
+<?php
+echo 'bar'
```

Comment:
Strange, that it didn't catch it as error. Maybe it's for `<?= 'foobar' ?>. I would still prefer a semicolon. Bug report anyone?